tests/test-http-api/test-config.js

```diff
@@ -0,0 +1,25 @@
+/* globals describe, it */
```

Comment:
There is a nicer way for this by the way: https://github.com/feross/standard#what-about-mocha-jasmine-qunit-etc